app/assets/stylesheets/components/page_specific/_promo.scss

```diff
@@ -12,13 +12,19 @@
   color: $color-blue-medium;
   margin: 0 0 $baseline-unit;
   position: relative;
+  display: inline-block;
+
+  &.promo__heading--blog {
+    display: block;
+  }
 
   @include respond-to($mq-m) {
     @include body(20, 24);
   }
 
-  .l-tool-promos__item & {
-    padding-right: $baseline-unit*3;
```

Comment:
Removing this causes a minor layout issue at certain screen widths: 
![image](https://cloud.githubusercontent.com/assets/14920201/14432823/b8da7fac-0003-11e6-979a-644829b75d9b.png)